mesheryctl/pkg/utils/manifest.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+type ManifestList struct {
+	SHA       string     `json:"sha"`
+	URL       string     `json:"url"`
+	Tree      []Manifest `json:"tree"`
+	Truncated bool       `json:"truncated"`
+}
+
+type Manifest struct {
+	Path string `json:"path"`
+	Mode string `json:"mode"`
+	Typ  string `json:"type"`
+	SHA  string `json:"sha"`
+	Size string `json:"size"`
+	URL  string `json:"url"`
+}
```

Comment:
for unmarshalling response from GitHub API call.

Reply:
Will adding omitempty fix the lint error shown below?

![Screenshot from 2021-02-06 13-55-36](https://user-images.githubusercontent.com/49474499/107113518-718df580-6885-11eb-92b2-50e1c23acab6.png)